<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arsenic Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="spinBox_clip">
        <property name="suffix">
         <string> sec</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_empty">
        <property name="text">
         <string>Empty clopboard after :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Langage (restart required)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="languageComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Password derivation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Iterations</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboMemory">
        <property name="currentText">
         <string notr="true">Interactive (64 mb)</string>
        </property>
        <item>
         <property name="text">
          <string notr="true">Interactive (64 mb)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Moderate (512 mb)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Sensitive (1000 mb)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="comboOps">
        <property name="currentText">
         <string notr="true">Interactive (1)</string>
        </property>
        <item>
         <property name="text">
          <string notr="true">Interactive (1)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Moderate (2)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Sensitive (3)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Memory Limit :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Files Encryption</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkAddEncrypted">
        <property name="text">
         <string>Add finished encrypted files to list</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Encryption type : </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboAlgo">
        <item>
         <property name="text">
          <string>ChaCha20Poly1305</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Serpent/GCM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Serpent/SIV</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Serpent/EAX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AES-256/GCM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AES-256/SIV</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AES-256/EAX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Triple Encryption</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <italic>true</italic>
       <bold>false</bold>
       <underline>false</underline>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>The decryption routine detect the right parameters automatically.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
